packages/mobile/e2e/src/DeepLinkSend.spec.js

```diff
@@ -1,7 +1,8 @@
 import HandleDeepLinkSend from './usecases/HandleDeepLinkSend'
 import { quickOnboarding } from './utils/utils'
+import { launchApp } from './utils/retries'
 
-describe('Deep link without account send', () => {
+describe('Given Deep Link', () => {
```

Comment:
is there a convention for how we name tests/specs? i see some of the top level specs are verbs but some are nouns and this one is "Given"...i don't have a preference but would be nice to have some consistent pattern to make the test reports really easy to read :) in the mobile unit tests the top level describe is usually the function or file name, so maybe this one could be `describe('Deep Link Send')` or something?

Reply:
I prefer the Given / When / Then behavior-driven development syntax for E2E tests, but this can also be a bit cumbersome for those not used to it.

Reply:
yeah i like the given/when/then too, although i'm more a fan of consistency than anything else :) i mention this here because i see most other specs look like `describe('Account', () => {})` or `describe('Send cUSD', () => {})` etc, so for me the version before this change was more consistent. if there is a plan to follow up this PR with an update of the naming conventions for other tests then that's all good, just wanted to flag it